manifests.yaml

```diff
@@ -59,6 +64,18 @@
       regexp: '^\s{2}replicas: 0$'
       replace: '  replicas: {{ initialWorkerCount }}'
 
+  - name: Find/replace MachineSet Name
+    replace:
+        path: openshift/99_openshift-cluster-api_worker-machineset-0.yaml
+        regexp: "{{ machineSetData.metadata.labels['machine.openshift.io/cluster-api-cluster'] }}-worker-0"
+        replace: "{{ ( custID + '-' + workerFlavor ) | replace('.','-') | replace('ocp-','') | replace('app-','') }}-worker"
+
+  - name: Find/replace serverMetadata Name
+    replace:
+        path: openshift/99_openshift-cluster-api_net2_worker-machineset-0.yaml
```

Comment:
Should this be openshift/99_openshift-cluster-api_worker-machineset-0.yaml ?

Reply:
Very much so - cheers. Luckily that serverMetadata name appears to do nothing. But still I will fix.